src/transformers/generation/utils.py

```diff
@@ -1594,6 +1592,20 @@ def generate(
                 )
             if generation_config.cache_implementation == "static":
                 model_kwargs["past_key_values"] = self._get_static_cache(batch_size, generation_config.max_length)
+            elif generation_config.cache_implementation == "quantized":
+                if not is_quanto_available():
+                    raise ImportError(
+                        "You need to install `quanto` in order to use KV cache quantization. "
+                        "Please install it via  with `pip install git+https://github.com/huggingface/quanto`"
+                    )
+
+                cache_config = (
+                    generation_config.cache_config
+                    if generation_config.cache_config is not None
+                    else QuantizedCacheConfig()
+                )
```

Comment:
```suggestion
```
can be removed if we pass generation_config.cache_config instead to the quantoQauntizedCache

Reply:
No, because `generation_config.cache_config` can be None if we are using the model's generation config and the user does not pass any cache config, In that case we have to init the Cache Config with default values ourselves. 

The one line below code will fail otherwise, if we pass in `None` as input